src/server.mjs

```
@@ -6,14 +6,21 @@ import cookieParser from "cookie-parser";
+//import "./strategies/local-strategy.mjs";
+import MongoStore from "connect-mongo";
+import "@dotenvx/dotenvx/config";
 
 const app = express();
 const port = process.env.PORT || 3000;
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.resolve(path.dirname(__filename), "..");
 
 //middlewares
+mongoose
+    .connect(`mongodb://localhost:27017/express`)
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)